src/instructlab/eval/mt_bench.py

```diff
@@ -260,10 +260,12 @@ def judge_answers(
             serving_gpus    Number of gpus allocated for serving.  Used to tune with max_workers=auto.  None indicates to use value specified in constructor.
 
         Returns:
+            overall_score   overall score from the evaluation
             qa_pairs        Question and answer pairs (with scores) from the evaluation
+            error_rate      percentage of questions dropped due to errors during evaluation
```

Comment:
If you do make another change to this commit, this is also missing for the mt_bench case above